js/cards.js

```diff
@@ -170,11 +179,10 @@ function setupHover() {
   // setup the initial event listeners for all cards
   let cardList = document.getElementsByClassName("card-image");
   for (let i = 0; i < cardList.length; i++) {
-    cardList[i].addEventListener("animationend", function () {
-      cardList[i].classList.remove("animated");
-      cardList[i].style.willChange = "auto";
-    });
+    cardList[i].addEventListener("animationend", removeAnimation);
+    cardList[i].parentElement.addEventListener("blur", removeAnimation);
```

Comment:
I read the blur event to mean "cancel the animation when you keyboard focus away" but that doesn't seem to happen, so I'm not 100% sure what purpose this event listener is? :0

Reply:
oh oop. I forgot to check if the target was the button element or not for removeAnimation (like in addAnimation). Thank you for catching this!